docs/dg/report.md

```diff
@@ -127,6 +127,6 @@ For ramps between the date ranges, the slices will be selected and it will be pr
 
 <!-- ==================================================================================================== -->
 
-## Segment view ([v_segment.js](https://github.com/reposense/RepoSense/blob/master/frontend/src/static/js/v_segment.js))
+## Segment view ([v_segment.vue](https://https://github.com/reposense/RepoSense/blob/master/frontend/src/components/v-segment.vue))
```

Comment:
I think there is an additional http here

Reply:
Fixed in a subsequent commit.